color-element.html

```diff
@@ -218,6 +220,12 @@
 
           _alphaContext: {
             type: CanvasRenderingContext2D
+          },
+          
+          isWithoutAlpha: {
+            type: Boolean,
+            value: false,
+            reflectToAttribute:true
```

Comment:
You don't seem to be using the fact that this property is being added to the host node anywhere. Unless you plan to use the attribute at some point for another purpose I'd remove this.